src/java.desktop/macosx/classes/com/apple/laf/AquaTabbedPaneUI.java

```diff
@@ -679,8 +691,16 @@ protected void paintContentBorder(final Graphics g, final int tabPlacement, fina
                 break;
         }
 
-        if (tabPane.isOpaque()) {
-            g.setColor(tabPane.getBackground());
+        if ((tabPane.isOpaque() || contentOpaque) && tabPane.getTabCount() > 0) {
```

Comment:
what is the difference between tabPane.isOpaque() and contentOpaque? Should isOpaque be set when contentOpaque is set?